packages/protocol/contracts/common/GasPriceMinimum.sol

```diff
@@ -47,51 +49,50 @@ contract GasPriceMinimum is Ownable, Initializable, UsingRegistry {
   function initialize(
     address _registryAddress,
     uint256 initialGas,
-    uint256 targetDensityNumerator,
-    uint256 targetDensityDenominator,
-    uint256 adjustmentSpeedNumerator,
-    uint256 adjustmentSpeedDenominator,
-    uint256 infrastructureFractionNumerator,
-    uint256 infrastructureFractionDenominator
+    uint256 _targetDensity,
+    uint256 _adjustmentSpeed,
+    uint256 _infrastructureFraction
```

Comment:
not sure, but maybe the word "Fraction" as something that has a numerator/denominator does not really apply